### PR TITLE
Fix activity message issue for the application deployed

### PR DESCRIPTION
**Pull request by TimothyAsirJeyasing**
https://bugzilla.redhat.com/show_bug.cgi?id=2234382

### CODE REVIEW

**Comment by TimothyAsirJeyasing** [comment]
@GowthamShanmugam Please review

**Comment by TimothyAsirJeyasing** [comment]
Screenshot from 2023-09-20 16-11-00

File: packages/mco/components/mco-dashboard/data-policy/cluster-app-card/argo-application-set.tsx [hunk 1]
```diff
@@ -49,6 +49,15 @@ const getCurrentActivity = (
       currentStatus,
       failoverCluster,
     });
+  } else if (
+    [DRPC_STATUS.Deployed, DRPC_STATUS.Deployed].includes(
```
Comment (bipuladh) [inline packages/mco/components/mco-dashboard/data-policy/cluster-app-card/argo-application-set.tsx:53]: Aren't these two the same?

Comment (TimothyAsirJeyasing) [reply packages/mco/components/mco-dashboard/data-policy/cluster-app-card/argo-application-set.tsx]: yes, will fix this

**Comment by bipuladh** [comment]
Please fix the build tests

**Review by bipuladh — APPROVED** [review]
/lgtm

**Comment by SanjalKatiyar** [comment]
> /hold

@GowthamShanmugam are we blocking this PR for some reason ??

**Comment by SanjalKatiyar** [comment]
is this ever discussed with UX @TimothyAsirJeyasing ??

**Comment by TimothyAsirJeyasing** [comment]
Yes this is discussed with UX, @Hadas David please confirm here if possible

**Comment by TimothyAsirJeyasing** [comment]
Yes, The changes confirmed with UX https://ibm-systems-storage.slack.com/archives/C06SS96EECT/p1715336088561499?thread_ts=1714389611.825559&cid=C06SS96EECT

**Comment by GowthamShanmugam** [comment]
Please close this PR, i think BZ is already closed for this

**Comment by nthomas-redhat** [comment]
@GowthamShanmugam , [BZ2234382](https://bugzilla.redhat.com/show_bug.cgi?id=2234382) is closed as NOTABUG. So I guess we can close this PR as well?

**Comment by GowthamShanmugam** [comment]
We can close this BZ

**Pull request closed without merging**